library/src/main/java/com/schibsted/spain/barista/BaristaScrollToAction.java

```diff
@@ -0,0 +1,46 @@
+package com.schibsted.spain.barista;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.support.test.espresso.action.ScrollToAction;
+import android.support.test.espresso.matcher.ViewMatchers;
+import android.support.v4.widget.NestedScrollView;
+import android.view.View;
+import android.widget.HorizontalScrollView;
+import android.widget.ScrollView;
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isAssignableFrom;
+import static android.support.test.espresso.matcher.ViewMatchers.isDescendantOfA;
+import static android.support.test.espresso.matcher.ViewMatchers.withEffectiveVisibility;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.anyOf;
+
+public class BaristaScrollToAction implements ViewAction {
```

Comment:
Yes. Actually, as it's an internal class, it's great to explain to other devs what it does. I mean, `BaristaScrollToAction` is the same than `AsdfghjkScrollToAction` 😂 As giving names is free, why not giving a good one? I suggest `NestedEnabledScrollToAction`. It's not the greatest name, but it expresses its intention.

As you note, if it changes in the future, let's say at 2042, the maintainers of the library will think in a better name for sure :joy: